.github/workflows/golang.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2024 NVIDIA CORPORATION
+# Copyright 2024-2025 NVIDIA CORPORATION
```

Comment:
Just keep the year the file was originally created

Reply:
done